app/serializers/api/participant_outcome_serializer.rb

```diff
@@ -0,0 +1,34 @@
+module API
+  class ParticipantOutcomeSerializer < Blueprinter::Base
+    identifier :ecf_id, name: :id
+    field(:type) { "participant-outcome" }
+
+    class AttributesSerializer < Blueprinter::Base
+      exclude :id
+
+      field(:state)
+      field(:completion_date)
+      field(:course_identifier) { |outcome| outcome.declaration.application.course.identifier }
+      field(:participant_id) { |outcome| outcome.user.ecf_id }
```

Comment:
There's a N+1 query here..
think we need to add user table to the [query](https://github.com/DFE-Digital/npq-registration/pull/1490/files#diff-b6c95ab3ddda088375890402e92d6a94b1525517a04cc1668c3b27826edc2e3aR40)

`application: [:course, :user],`

Reply:
Good spot, I've updated 👍